packages/mermaid/src/diagrams/gantt/ganttRenderer.js

```diff
@@ -771,6 +798,25 @@ export const draw = function (text, id, version, diagObj) {
     }
   }
 
+  /**
+   * @param theSidePad
+   * @param theTopPad
+   * @param w
+   * @param h
+   */
+  function _drawDate(theSidePad, theTopPad, w, h) {
+    const todayG = svg.append('g').attr('class', 'today');
+    const today = new Date();
+    const todayLine = todayG.append('line');
+
+    todayLine
+      .attr('x1', timeScale(today) + theSidePad)
+      .attr('x2', timeScale(today) + theSidePad)
+      .attr('y1', conf.titleTopMargin)
+      .attr('y2', h - conf.titleTopMargin)
+      .attr('class', 'today');
+  }
```

Comment:
Is this function still needed?

Reply:
I'm asking because it seems it did not exist beforehand and I couldn't find a single usage of this method in the codebase.